database/src/main/resources/db/migration/V7__add_timer_heart_rate.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE stream_metadata ADD COLUMN heart_rates int[];
+
+UPDATE stream_metadata SET heart_rates = '{}';
+
+CREATE TABLE timers (
+    id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+    start_time TEXT,
+    end_time TEXT
```

Comment:
I got date fields working with incentives. It would be better to use those instead. I also learned that OffsetDateTime should be used, instead of Date in the models.